benefits/static/css/admin/styles.css

```diff
@@ -5,13 +5,14 @@
 html[data-theme="light"],
 :root {
   --primary: #ffffff;
-  --secondary: var(--dark-color);
-  --link-fg: #ffffff;
```

Comment:
This was making all links white. This is way too big of an override. Removed it and just added a `text-white` to the button in question (New enrollment button) instead.